go.mod

```diff
@@ -19,6 +19,8 @@ require (
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/kylelemons/godebug v0.0.0-20160406211939-eadb3ce320cb
 	github.com/matttproud/golang_protobuf_extensions v1.0.1
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
```

Comment:
You need to use Go 1.13 (or not commit this change).

Reply:
ok.